src/jit/emit.h

```diff
@@ -765,7 +765,7 @@ class emitter
         // x86:   24 bits
         // amd64: 16 bits
         // arm:   14 bits
-        // arm64: 14 bits
+        // arm64: 13 bits
 
         unsigned _idSmallCns : ID_BIT_SMALL_CNS;
```

Comment:
The size of the small constant area is something to keep an eye on.
We like it to be at least 12 bits which it is, but more is always better.